src/Markup/Avalonia.Markup/Data/MultiBinding.cs

```diff
@@ -80,12 +80,13 @@ public MultiBinding()
             if (!string.IsNullOrWhiteSpace(StringFormat) &&
                 (targetType == typeof(string) || targetType == typeof(object)))
             {
-                converter = new StringFormatMultiValueConverter(StringFormat, converter);
+                converter = new StringFormatMultiValueConverter(StringFormat!, converter);
```

Comment:
Do we really need "!" operator here for StringFormat? Or is it because of old target framework (netstandard), where IsNullOrWhiteSpace is not annotated with attributes?

Reply:
Yeah I think this is a problem with targeting netstandard2.0 where `IsNullOrWhiteSpace` isn't correctly annotated: https://github.com/dotnet/roslyn/issues/37995#issuecomment-522483947